style.css

```diff
@@ -1,4 +1,623 @@
+    font-weight: bold;
+    line-height: 72px;
+    margin-top: 60px;
+  }
+
+  .collection__description {
+    margin-bottom: 2.5vw;
+  }
+
+  .collection__btn {
+    font-size: 12px;
+    margin-right: 3vw;
+  }
+
+  .range__link {
+    font-size: 12px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .header__logo {
+    margin-right: 7vw;
+  }
+
+  .language {
+    display: none;
+  }
+
+  .collection {
+    padding-top: 45px;
+  }
+
+  .collection__title {
+    font-size: 42px;
+    line-height: 43px;
+    margin-bottom: 2vw;
+    margin-top: 20px;
+  }
+
+  .collection__description {
+    font-size: 13px;
+    line-height: 26px;
+    margin-bottom: 2vw;
+  }
+
+  .collection__btn {
+    font-size: 11px;
+    margin-right: 2.5vw;
+  }
+
+  .range__link {
+    color: #000;
+    font-size: 11px;
+    margin-right: 1vw;
+  }
+
+  .scroll {
+    display: none;
+  }
 }
 
+@media (max-width: 880px) {
+  .collection__img {
+  	height: 35%;
+  	width: 35%;
+  }
+
+  .collection {
+  	padding-right: 11vw;
+  }
+
+  .collection__img {
+    margin-bottom: 7vw;
+}
+
+  .collection__title {
+  	margin-top: 7vw;
+  }
+
+  .collection__description {
+    font-size: 15px;
+    line-height: 30px;
+    margin-bottom: 3vw;
+  }
+
+  .collection__btn {
+  	font-size: 12px;
+  	margin-right: 4vw;
+  }
+
+  .range__link {
+  	font-size: 12px;
+  }
+}
+
+@media (min-width: 568px) and (max-width: 767px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  html,
+  body {
+    margin: 0 auto;
+    max-width: 767px;
+    min-width: 568px;
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .main-page {
+  	background: none;
+  	padding: 0;
+  	margin: 0;
+  }
+
+  .header,
+  .collection {
+    width: 100%;
+  }
+
+  .header {
+    background-color: #fff;
+    display: flex;
+    font-size: 0;
+    height: 69px;
+    justify-content: flex-start;
+  }
+
+  .header__logo {
+    background-image: url('./images/LOCO.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: fixed;
+    display: flex;
+    height: 40px;
+    margin-left: 26px;
+    transform: translateY(-2px);
+    width: 98px;
+  }
+
+  .header-menu {
+    background-color: #ededed;
+    height: 69px;
+    margin: 0;
+    right: 0;
+    width: 70px;
+  }
+
+  .header-menu__line::before {
+    content: '';
+    transform: translateY(-9px);
+  }
+
+  .header-menu__line,
+  .header-menu__line::before {
+    display: block;
+    height: 3px;
+    left: 50%;
+    margin-left: -14px;
+    top: 52%;
+    width: 27px;
+  }
+
+  .language,
+  .collection__img,
+  .collection__description,
+  .collection__btn,
+  .range,
+  .socials-list,
+  .scroll {
+    display: none;
+  }
+
+  .collection {
+    background-image: url('./images/pete-bellis-432867-unsplash-mob.png');
+    background-position: center center;
+    background-size: cover;
+    height: 870px;
+    object-fit: cover;
+    padding: 0;
+    width: 100%;
+  }
+
+  .collection-content {
+  	width: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)